solr/solr-ref-guide/src/circuit-breakers.adoc

```diff
@@ -27,23 +27,16 @@ If circuit breakers are enabled, requests may be rejected under the condition of
 It is up to the client to handle this error and potentially build a retrial logic as this should ideally be a transient situation.
 
 == Circuit Breaker Configurations
-All circuit breaker configurations are listed in the `<circuitBreaker>` tags in `solrconfig.xml` as shown below:
+All circuit breaker configurations are listed as independent `<circuitBreaker>` entries in `solrconfig.xml` as shown below:
 
 [source,xml]
 ----
-<circuitBreaker class="solr.CircuitBreakerManager" enabled="true">
-  <!-- All specific configs in this section -->
+<circuitBreaker class="org.apache.solr.util.circuitbreaker.CPUCircuitBreaker" enable="true">
+  <int  name="threshold">75</int>
 </circuitBreaker>
 ----
 
-The `enabled` attribute controls the global activation/deactivation of circuit breakers.
-If this flag is disabled, all circuit breakers will be disabled globally.
-Per circuit breaker configurations are specified in their respective sections later.
-
-This attribute acts as the highest authority and global controller of circuit breakers.
-For using specific circuit breakers, each one needs to be individually enabled in addition to this flag being enabled.
-
-`CircuitBreakerManager` is the default manager for all circuit breakers that should be defined in the tag unless the user wishes to use a custom implementation.
+Each circuit breaker can be independently enabled/disabled using the corresponding flag.
```

Comment:
I'm confused about this because it looks like we don't read the flag anywhere anymore.

Reply:
2/2: Alternative wording might be this:

```suggestion
Each circuit breaker plugin can be independently enabled/disabled using the `enable` attribute.
```

Or to remove the sentence and the `enable="true"` mentions in the examples if/since by defaults plugins would presumably be enabled?

Another possibility would be

```
<circuitBreaker class="Foobar" enable="${circuitBreaker.foobar.enable:true}">
  ...
</circuitBreaker>
```

style examples on the (unconfirmed) assumption that this system property usage is supported for these plugins, and this would be a way for users to change the behaviour of an instance without a config change.

Reply:
I would disagree on the proposed XML format -- seems too clumsy to me. We could remove the enable flag altogether, although it exists for quick experimentation (toggle on/off) when using circuit breakers.